obspy/core/tests/test_event.py

```diff
@@ -675,3 +675,9 @@ def test_setting_nans_or_inf_fails(self):
 
         with pytest.raises(ValueError, match='is not a finite'):
             o.latitude = float('-inf')
+
+    def test_issue3105(self):
```

Comment:
For consistency with naming the test functions, it may be good to give this a more explicit name (even though there is indeed one other test function here with a name like test_issue...).  And / or a comment could explicitly describe the bug that this function tests.

Reply:
I dont care either way, I think we have quite some test routine names like that across code base, and it's just a test function anyway after all =)